section/algorithms.html

```diff
@@ -119,6 +119,50 @@ <h3>Outgoing payment and quote grant request</h3>
             <li>
                 If |authServer| [=url/scheme=] is not `https`, return failure.
             </li>
+            <li>
+                <p>Construct a {{ GrantRequest }} |grantRequest| as follows:</p>
+                <ol>
+                    <li>
+                        Set |grantRequest|'s {{GrantRequest/client}} to <var>walletAddressDetails</var>'s {{WalletAddressDetails/url}} property.
+                    </li>
+                    <li>
+                        Set |grantRequest|'s {{GrantRequest/interact}} to a new {{ Interact }} with {{Interact/start}} set to `["redirect"]`.
+                    </li>
+                    <li>
+                        Let |quoteAccess| be a new {{ Access }} object with:
+                        <ul>
+                            <li>{{Access/type}} set to `"quote"`</li>
+                            <li>{{Access/actions}} set to `["create"]`</li>
+                        </ul>
+                    </li>
+                    <li>
+                        Let |debitAmount| be a new {{ Amount }} object with:
+                        <ul>
+                            <li>{{Amount/value}} set to |budget|</li>
```

Comment:
`budget` is a user provided number here, but `value` accepts number as a string, as per `assetCode`

```ts
const value = Math.floor(Number.parseFloat(budget) * 10 ** assetScale).toString()
// $1.5 with assetScale=2 means value="150"
```